frontend/src/components/VariantListPage/VariantListPage.tsx

```diff
@@ -369,6 +396,62 @@ const useVariantListAnnotation = (variantList: VariantList) => {
     [variantList]
   );
 
+  const saveTaggedGroups = useCallback(
+    (taggedGroups) => {
+      patch(`/variant-lists/${variantList.uuid}/shared-annotation/`, {
+        tagged_groups: {
+          A: {
+            displayName: String(taggedGroups.A.displayName),
+            variantList: Array.from(taggedGroups.A.variantList),
+          },
+          B: {
+            displayName: String(taggedGroups.B.displayName),
+            variantList: Array.from(taggedGroups.B.variantList),
+          },
+          C: {
+            displayName: String(taggedGroups.C.displayName),
+            variantList: Array.from(taggedGroups.C.variantList),
+          },
+          D: {
+            displayName: String(taggedGroups.D.displayName),
+            variantList: Array.from(taggedGroups.D.variantList),
+          },
+        },
+      })
+        .then(() => {
+          toast({
+            title: "Saved tags",
+            status: "success",
+            duration: 1000,
+            isClosable: true,
+          });
+        })
+        .catch((error) => {
+          toast({
+            title: "Unable to save groups",
+            description: renderErrorDescription(error),
+            status: "error",
+            duration: 10000,
+            isClosable: true,
+          });
+        });
+    },
```

Comment:
It could be a good idea to wrap this in a `debounce` to try and batch a network request, as it stands every time you change one thing we make a round trip to the network. Not a big deal -- but if people are likely going to tag a few variants, a debounce with a 3-5 second wait period could be a way to save on some network trips.

`saveSelectedVariants` uses `lodash`'s debounce function, for example.